src/main/java/uk/nhs/ctp/service/CdssSupplierService.java

```diff
@@ -90,6 +91,7 @@ public CdssSupplier createCdssSupplier(NewCdssSupplierDTO newCdssSupplierDTO) {
     cdssSupplier.setBaseUrl(newCdssSupplierDTO.getBaseUrl());
     cdssSupplier.setInputDataRefType(newCdssSupplierDTO.getInputDataRefType());
     cdssSupplier.setInputParamsRefType(newCdssSupplierDTO.getInputParamsRefType());
+    cdssSupplier.setSupportedVersion(SupportedVersion.TWO); //TODO: CDSCT-221 Get from UI
```

Comment:
Should this default to the `1.1` spec? (based on the title of CDSCT-115 saying "initially 1.1 only")

Reply:
sure, that'll get implemented in the actual story to set it, this is more of a placeholder